app.py

```
@@ -0,0 +1,249 @@
+            lc2.metric("Stop", f"${result['stop']:.2f}")
+            lc3.metric("Target 1", f"${result['target1']:.2f}")
+            lc4.metric("Target 2", f"${result['target2']:.2f}" if result["target2"] else "N/A")
+            lc5.metric("Position %", f"{result['position_size_pct']}%")
+
+        # --- Meta ---
+        st.markdown("#### Multi-Timeframe Summary")
+        mc1, mc2, mc3, mc4 = st.columns(4)
+        mc1.info(f"**1h Trend:** {result['meta']['hourly_trend']}")
+        mc2.info(f"**News Impact:** {result['meta']['news_impact']}")
+        mc3.info(f"**News Conflict:** {'Yes' if result['meta']['news_sentiment_conflict'] else 'No'}")
+        mc4.info(f"**15m Setup:** confirmed" if action != "Hold" else f"**15m Setup:** not confirmed")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)